spec/helpers/feedbacks_helper_spec.rb

```diff
@@ -17,7 +17,7 @@ class Klass
       end
 
       it 'should not include invitations that already have submitted feedback associated with them' do
-        feedback = create(:submitted_feedback, user: user, review: invitations.first.review)
+        create(:submitted_feedback, user: user, review: invitations.first.review)
         klass.open_requests(user).should == 4
```

Comment:
4 is a weird magic number here.  Can we do something with `expect(create_feedback)to change(open_request count)`?